pkg/config/config.go

```diff
@@ -161,8 +162,17 @@ func unmarshalSinks(sinksIface interface{}) (sink.Sinks, error) {
 			if err != nil {
 				return nil, errors.Wrap(err, "unable to authenticate travis API")
 			}
-
 			sinks = append(sinks, &sink.TravisCiSink{BaseSink: sink.BaseSink{KeyToName: keyToName}, RepoSlug: sinkMapStr["repo_slug"], Client: client})
+
+		case sink.KindCircleCi:
+			if err = validate(sinkMapStr, "account", "repo"); err != nil {
+				return nil, errors.Wrap(err, "missing keys in circle CI sink config")
+			}
+			client := &circleci.Client{Token: os.Getenv("CIRCLECI_AUTH_TOKEN")}
```

Comment:
Correct me if I am wrong, but there is no way for writing to circle to work without this env var, right?

If that's the case we should use [LookupEnv](https://golang.org/src/os/env.go?s=3235:3276#L102) and return a good error if the env variable is not set.